### PR TITLE
Make the calendar configurable

### PR DESCRIPTION
**Pull request by rowanmanning**
You can now choose from a list of predefined calendars when creating a
campaign. For now this includes Harptos, Odreian, and Tide.

Odreian is the default at the moment as I don't want this PR to affect
existing campaigns. Because there aren't very many at the moment we
could just manually add the calendar property to each of them and then
set the default to Harptos.

I've added a version number to each of the calendar IDs so that we can
iterate on them later if we want to without old campaign dates being
knocked out.

Matt: I expect that there's loads wrong with this because I'm not at all
familiar with Meteor or React.

### CODE REVIEW

File: client/blocks/quests/index.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ const Objective = connectObjective(({
 	}
 
 	{objective.completed && <Completed>
-		{new OdreianDate(objective.completedDate).format`${'llll'}`}
+		{new OdreianDate(objective.completedDate).format`${'h'}:${'mm'}${'a'}, ${'dddd'}, ${'Do'} of ${'MM'}, ${'YY'}`}
```
Comment (rowanmanning) [inline client/blocks/quests/index.js]: Note: this is hardcoded for now until I tackle rowanmanning/dream-date#9

**Comment by apaleslimghost** [comment]
i'll take a look through this tomorrow. a sidenote, we will need some migrations at some point, e.g. #80 is going to make literally all the cards inaccessible to anybody. of course it's small enough we could fix it with ✨ *manual database editing* ✨ for now

File: client/pages/home.js [hunk 2]
```diff
@@ -29,6 +30,12 @@ export default withCampaignData(({campaigns, createCampaign}) => <ul>
 	<li>
 		<form onSubmit={createCampaign}>
 			<input placeholder='Campaign' name='title' />
+			<select name='calendar'>
```
Comment (apaleslimghost) [inline client/pages/home.js:33]: eventually this will want to move to a campaign settings page but it's fine here for now

**Review by apaleslimghost — DISMISSED** [review]
this is actually perfect. there's not a single thing i would have done differently 👌

File: client/blocks/time/display.js [hunk 3]
```diff
@@ -32,7 +32,7 @@ const ornaments = [
 	'h', 'f', 'a', 't', 'n', 'c', 'o', 'p', 'e', 'r', 'k', 'l'
 ];
 
-const OrnamentedMonth = ({date}) => <Ornamented ornament={ornaments[date.monthIndex]} large>
+const OrnamentedMonth = ({date}) => <Ornamented ornament={ornaments[Math.min(date.monthIndex, 11)]} large>
```
Comment (apaleslimghost) [inline client/blocks/time/display.js:35]: so, the ornaments were kind of chosen to match the odreian month names. maybe at some point we want to do configurable display styles for the time block?

**Review by apaleslimghost — APPROVED** [review]

File: client/blocks/time/display.js (hunk 3 above)
Comment (apaleslimghost) [reply client/blocks/time/display.js:35]: screen shot 2018-04-20 at 21 54 01

Comment (rowanmanning) [reply client/blocks/time/display.js:35]: Yeah having them configurable would be nice. They'll still work the same way for calendars with 12-month years

**Comment by apaleslimghost** [comment]
@rowanmanning is this ready to merge?

**Comment by rowanmanning** [comment]
Yup I reckon so!

**Pull request merged**